src/pages/Order/Order.js

```diff
@@ -0,0 +1,286 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+const Order = () => {
+  const [orderList, setOrderList] = useState([]);
+  const [openList, setOpenList] = useState(false);
+  const [orderInfo, setOrderInfo] = useState([]);
+  const tokenOrder = localStorage.getItem('token', orderList.token);
+  const orderArray = orderList[0];
+
+  useEffect(() => {
+    fetch('/data/orderData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setOrderList(data);
+      });
+  }, []);
+
+  const [isChecked, setIsChecked] = useState('');
+
+  const handleChecked = e => {
+    if (isChecked === '') {
+      setIsChecked(true);
+    } else if (isChecked === false) {
+      setIsChecked(true);
+    } else {
+      setIsChecked('');
+    }
+  };
+
+  const checkPart =
+    orderArray &&
+    orderArray.part.map(({ id }) => {
+      return { ...orderInfo, id };
+    });
+
+  const onclickToBuy = () => {
+    isChecked && tokenOrder
```

Comment:
```suggestion
        if (!isChecked) return setIsChecked(false);
```

해당 로직은 얼리리턴 적용해서 구현해 볼 수 있겠습니다
3항 연산자는 안 써도 될 것 같아요